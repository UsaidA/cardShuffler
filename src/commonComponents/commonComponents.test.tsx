import renderer from 'react-test-renderer';
import Navbar from './navBar';
import PlayingCard from './Playingcard';

it('Navbar Component is rendered correctly', () => {
  const tree = renderer
    .create(<Navbar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const tree = renderer
      .create(<PlayingCard cardNumber="A" cardSuit="Hearts" />)
      .toJSON();
  
    expect(tree).toMatchSnapshot();
  });