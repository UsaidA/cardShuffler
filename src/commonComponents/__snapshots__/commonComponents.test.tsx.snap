// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component is rendered correctly 1`] = `
<nav
  className="navbar fixed-top navbar-expand-lg navbar-light"
  style={
    Object {
      "backgroundColor": "#bcbcbc83",
    }
  }
>
  <div
    className="container"
  >
    <div>
      <button
        className="back"
        onClick={[Function]}
      >
        BACK!
      </button>
    </div>
    <div
      className="collapse navbar-collapse"
      id="navbarNav"
    >
      <ul
        className="navbar-nav ms-auto"
      />
    </div>
  </div>
</nav>
`;

exports[`renders correctly 1`] = `<div />`;
